pkg/admission/validate_noobaa.go

```diff
@@ -0,0 +1,45 @@
+package admission
+
+import (
+	"github.com/sirupsen/logrus"
+	admissionv1 "k8s.io/api/admission/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// NewNoobaaValidator initializes a NoobaaAccountValidator to be used for loading and validating a noobaaaccount
```

Comment:
I Think it was not meant to be NoobaaAccountValidator.
```suggestion
// NewNoobaaValidator initializes a NoobaaValidator to be used for loading and validating a noobaa
```

Reply:
Fixed ✅